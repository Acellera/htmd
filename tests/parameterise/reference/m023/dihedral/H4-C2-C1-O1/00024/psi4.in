set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -1.070890	 -1.366565	 0.180696
O	 -1.422729	 -0.573567	 -0.196946
C	 -0.731407	 0.539437	 0.309156
C	 0.680193	 0.672443	 -0.251532
H	 -1.319765	 1.404033	 0.030086
H	 -0.700012	 0.506517	 1.397677
C	 1.591892	 -0.504292	 0.097471
H	 0.615328	 0.777117	 -1.330644
H	 1.113807	 1.593439	 0.132246
H	 1.225600	 -1.431248	 -0.333548
H	 2.594621	 -0.347452	 -0.285806
H	 1.668575	 -0.639190	 1.173661

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		1 2 3 4
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "12  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
