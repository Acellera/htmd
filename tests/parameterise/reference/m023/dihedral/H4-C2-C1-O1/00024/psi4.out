    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:24:23 2016

    Process ID:   9422
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -1.070890	 -1.366565	 0.180696
O	 -1.422729	 -0.573567	 -0.196946
C	 -0.731407	 0.539437	 0.309156
C	 0.680193	 0.672443	 -0.251532
H	 -1.319765	 1.404033	 0.030086
H	 -0.700012	 0.506517	 1.397677
C	 1.591892	 -0.504292	 0.097471
H	 0.615328	 0.777117	 -1.330644
H	 1.113807	 1.593439	 0.132246
H	 1.225600	 -1.431248	 -0.333548
H	 2.594621	 -0.347452	 -0.285806
H	 1.668575	 -0.639190	 1.173661

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		1 2 3 4
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "12  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-1-3-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:24:23 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H         -1.069079520348    -1.363525215820     0.185975609801     1.007825032070
           O         -1.420918520348    -0.570527215820    -0.191666390199    15.994914619560
           C         -0.729596520348     0.542476784180     0.314435609801    12.000000000000
           C          0.682003479652     0.675482784180    -0.246252390199    12.000000000000
           H         -1.317954520348     1.407072784180     0.035365609801     1.007825032070
           H         -0.698201520348     0.509556784180     1.402956609801     1.007825032070
           C          1.593702479652    -0.501252215820     0.102750609801    12.000000000000
           H          0.617138479652     0.780156784180    -1.325364390199     1.007825032070
           H          1.115617479652     1.596478784180     0.137525609801     1.007825032070
           H          1.227410479652    -1.428208215820    -0.328268390199     1.007825032070
           H          2.596431479652    -0.344412215820    -0.280526390199     1.007825032070
           H          1.670385479652    -0.636150215820     1.178940609801     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.48044  B =      0.17035  C =      0.14280 [cm^-1]
  Rotational constants: A =  14403.12729  B =   5106.91089  C =   4281.07681 [MHz]
  Nuclear repulsion =  133.127898000174781

  Charge       = 0
  Multiplicity = 1
  Electrons    = 34
  Nalpha       = 17
  Nbeta        = 17

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 40
    Number of basis function: 76
    Number of Cartesian functions: 76
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         76      76       0       0       0       0
   -------------------------------------------------------
    Total      76      76      17      17      17       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 168
    Number of basis function: 524
    Number of Cartesian functions: 524
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 5.2552180722E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -155.87784551675421   -1.55878e+02   8.72177e-02 
   @DF-RHF iter   2:  -120.79524346099544    3.50826e+01   8.58790e-02 DIIS
   @DF-RHF iter   3:  -170.03704910356672   -4.92418e+01   5.18066e-02 DIIS
   @DF-RHF iter   4:  -169.60984220046558    4.27207e-01   6.40641e-02 DIIS
   @DF-RHF iter   5:  -192.87110137969975   -2.32613e+01   9.63356e-03 DIIS
   @DF-RHF iter   6:  -193.05066202068818   -1.79561e-01   5.03524e-03 DIIS
   @DF-RHF iter   7:  -193.10834522165416   -5.76832e-02   6.32886e-04 DIIS
   @DF-RHF iter   8:  -193.10986862390601   -1.52340e-03   1.35909e-04 DIIS
   @DF-RHF iter   9:  -193.10996547378156   -9.68499e-05   3.17135e-05 DIIS
   @DF-RHF iter  10:  -193.10997068510952   -5.21133e-06   1.12704e-05 DIIS
   @DF-RHF iter  11:  -193.10997142220631   -7.37097e-07   3.23566e-06 DIIS
   @DF-RHF iter  12:  -193.10997146487509   -4.26688e-08   1.02528e-06 DIIS
   @DF-RHF iter  13:  -193.10997147089134   -6.01625e-09   1.82913e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.553254     2A    -11.268043     3A    -11.218724  
       4A    -11.217525     5A     -1.360174     6A     -1.051019  
       7A     -0.927668     8A     -0.809139     9A     -0.696738  
      10A     -0.636238    11A     -0.601922    12A     -0.588605  
      13A     -0.538025    14A     -0.510097    15A     -0.493507  
      16A     -0.468845    17A     -0.429608  

    Virtual:                                                              

      18A      0.215803    19A      0.262741    20A      0.289388  
      21A      0.292854    22A      0.312378    23A      0.323081  
      24A      0.345973    25A      0.355515    26A      0.394346  
      27A      0.423944    28A      0.450374    29A      0.716871  
      30A      0.726482    31A      0.757226    32A      0.790409  
      33A      0.825491    34A      0.905733    35A      0.937862  
      36A      0.969381    37A      1.039878    38A      1.086609  
      39A      1.137422    40A      1.150208    41A      1.160248  
      42A      1.190107    43A      1.201420    44A      1.214522  
      45A      1.236511    46A      1.248529    47A      1.303483  
      48A      1.344716    49A      1.456335    50A      1.544186  
      51A      1.690723    52A      1.724853    53A      1.830860  
      54A      1.900969    55A      1.968192    56A      2.019795  
      57A      2.073124    58A      2.191561    59A      2.273555  
      60A      2.312091    61A      2.368696    62A      2.457398  
      63A      2.528357    64A      2.590904    65A      2.662397  
      66A      2.697047    67A      2.715735    68A      2.787256  
      69A      2.885139    70A      2.989415    71A      3.047772  
      72A      3.208073    73A      4.181673    74A      4.586094  
      75A      4.743115    76A      4.904264  

    Final Occupation by Irrep:
              A 
    DOCC [    17 ]

  Energy converged.

  @DF-RHF Final Energy:  -193.10997147089134

   => Energetics <=

    Nuclear Repulsion Energy =            133.1278980001747811
    One-Electron Energy =                -525.7374570307799786
    Two-Electron Energy =                 199.4995875597138024
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -193.1099714708913666

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     3.8759      Y:     0.4857      Z:     0.9427

  Electronic Dipole Moment: (a.u.)
     X:    -3.3051      Y:    -0.5021      Z:    -0.5189

  Dipole Moment: (a.u.)
     X:     0.5708      Y:    -0.0165      Z:     0.4238     Total:     0.7112

  Dipole Moment: (Debye)
     X:     1.4509      Y:    -0.0419      Z:     1.0773     Total:     1.8076


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-1-3-4.cx1.hpc.ic.ac.uk at Tue Jul  5 16:24:26 2016
Module time:
	user time   =       2.84 seconds =       0.05 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          3 seconds =       0.05 minutes
Total time:
	user time   =       2.84 seconds =       0.05 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          3 seconds =       0.05 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     3.8759      Y:     0.4857      Z:     0.9427

  Electronic Dipole Moment: (a.u.)
     X:    -3.3051      Y:    -0.5021      Z:    -0.5189

  Dipole Moment: (a.u.)
     X:     0.5708      Y:    -0.0165      Z:     0.4238     Total:     0.7112

  Dipole Moment: (Debye)
     X:     1.4509      Y:    -0.0419      Z:     1.0773     Total:     1.8076

  Quadrupole Moment: (Debye Ang)
    XX:   -30.3949     YY:   -23.9278     ZZ:   -26.7404
    XY:    -0.4778     XZ:    -1.4951     YZ:    -0.9106

  Traceless Quadrupole Moment: (Debye Ang)
    XX:    -3.3739     YY:     3.0933     ZZ:     0.2806
    XY:    -0.4778     XZ:    -1.4951     YZ:    -0.9106

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.28004  0.28004  0.00000  0.43991
       2     O     4.37037  4.37037  0.00000 -0.74075
       3     C     2.99704  2.99704  0.00000  0.00593
       4     C     3.16048  3.16048  0.00000 -0.32096
       5     H     0.41442  0.41442  0.00000  0.17117
       6     H     0.43224  0.43224  0.00000  0.13552
       7     C     3.25053  3.25053  0.00000 -0.50106
       8     H     0.41410  0.41410  0.00000  0.17180
       9     H     0.42313  0.42313  0.00000  0.15375
      10     H     0.42141  0.42141  0.00000  0.15718
      11     H     0.41375  0.41375  0.00000  0.17249
      12     H     0.42249  0.42249  0.00000  0.15502

   Total alpha = 17.00000, Total beta = 17.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
